app/views/partners/requests/new.html.erb

```diff
@@ -42,7 +42,7 @@
                 </tr>
                 </thead>
                 <tbody class='fields' data-request-item-target="addDest">
-                  <%= render 'item_request', form: form, object: @partner_request.item_requests.build %>
+                  <%= render 'item_request', form: form %>
```

Comment:
Wait, why was this taken out?

Reply:
I added that initially. Not sure if it was in this PR but it resulted in inputs not being preserved. I had a misunderstanding about why it was removing an input field. Changing it is correct behavior.

Reply:
Sorry, can you clarify - is it correct to _have_ the `object: ...` part of the code, or is it correct to _not have_ it?

Reply:
Correct version: **no** `object: ...`. 

`object: ....` causes the form to not save user input if an update fails. It clears the form instead.